pkg/kotsadm/objects/images.go

```diff
@@ -37,6 +41,7 @@ func GetAdminConsoleImages(deployOptions types.DeployOptions) map[string]string
 		"kotsadm-migrations": fmt.Sprintf("%s/kotsadm-migrations:%s", kotsadmversion.KotsadmRegistry(deployOptions.RegistryConfig), kotsadmversion.KotsadmTag(deployOptions.RegistryConfig)),
 		"kotsadm":            fmt.Sprintf("%s/kotsadm:%s", kotsadmversion.KotsadmRegistry(deployOptions.RegistryConfig), kotsadmversion.KotsadmTag(deployOptions.RegistryConfig)),
 		"minio":              minioImage,
+		"minio-client":       minioClientImage,
```

Comment:
this should also be added in the `GetOriginalAdminConsoleImages` function below.